src/controllers/post.js

```diff
@@ -329,3 +330,59 @@ export const deleteLike = async (req, res) => {
     throw err
   }
 }
+
+export const createCommentLike = async (req, res) => {
```

Comment:
No problem with the way you've implemented this but it's a good topic for a whiteboarding / discussion session in a dev team. The question about post / comment reactions is often "Are we updating comment data or do we treat it like a separate entity in the application?"

Some engineers argue that a REST API is an abstraction so it would be preferable to treat reactions as part of the comment / post model at an interface level. So adding / removing a like to a comment should respond with the comment / post that received the like, including the updated like count.

I'm fine with both approaches, just something to think about while you have a coffee